packages/discord.js/typings/index.d.ts

```diff
@@ -4393,6 +4393,12 @@ export class GuildStickerManager extends CachedManager<Snowflake, Sticker, Stick
   public fetchUser(sticker: StickerResolvable): Promise<User | null>;
 }
 
+export interface ModifyGuildMemberRolesOptions {
```

Comment:
Is this the right place to put this? What is the preference for ordering in the this typings file